ui/src/components/AppHeader.js

```diff
@@ -1,56 +1,69 @@
 import React from 'react';
 import { makeStyles } from '@material-ui/core/styles';
 
-import AppBar from '@material-ui/core/AppBar';
 import IconButton from '@material-ui/core/IconButton';
-import Toolbar from '@material-ui/core/Toolbar';
-import Typography from '@material-ui/core/Typography';
 import MenuIcon from '@material-ui/icons/Menu';
 
 const useStyles = makeStyles(theme => ({
-  appBar: ({ drawerWidth }) => ({
+  appBar: {
```

Comment:
the type of `appBar` is changing from a function to a record... that's on purpose?
I'm not familiar with `@material-ui/core`.

My normal critieria for approving a PR is: would I be happy to maintain this code?
I'm afraid I would have to do quite a bit of study to understand how this stuff works.

A lower bar is: do I see any critical problems in the code?
I don't _know_ that this is wrong, so I could just let it go, but I would need a better sense from the team about what our goal is here.

Reply:
as we discussed, the `makeStyles` API supports both functions and records, so yes, this was on purpose

Reply:
Yea that's just how makeStyles works, we're no longer depending on the theme for the `appBar` styles so we can use a record instead of a function